RadarSDK/Radar.m

```diff
@@ -711,6 +711,53 @@ + (void)getMatrixFromOrigins:(NSArray<CLLocation *> *_Nonnull)origins
                                         }];
 }
 
++ (void)sendEvent:(NSString *)name
+     withMetadata:(NSDictionary *_Nullable)metadata
+completionHandler:(RadarSendEventCompletionHandler)completionHandler {
+
+    // TODO(jsani): check if permissions are enabled for events that can occur before location permission prompts
+    // if not, send the request with location = NULL
```

Comment:
there is a valid use case here of sending events without a location, but that pattern doesn't exist anywhere else in this file - would it make sense to have a local instance of `RadarPermissionsHelper` here? `RadarLocationManager` doesn't expose `locationAuthorizationStatus` like the permissions helper does https://github.com/radarlabs/radar-sdk-ios/blob/develop/RadarSDK/RadarPermissionsHelper.h#L15

/cc @jrtibbetts @nickpatrick